apps/api/src/domain/models/shared/events/base-event.entity.ts

```diff
@@ -47,6 +49,10 @@ export abstract class BaseEvent<
         return isDeepStrictEqual(this.payload[AGGREGATE_COMPOSITE_IDENTIFIER], compositeIdentifier);
     }
 
+    public getPayload() {
+        return this.payload;
+    }
+
     public toDTO<T extends BaseEvent>(this: T): DTO<this> {
```

Comment:
Oh interesting. I didn't realize we already had a mapping layer. Maybe we want to implement this differently for each view so we can remove hidden props.